### PR TITLE
Make Database Engine Selectable:

### PR DESCRIPTION
**Pull request by avattathil**
- Defaults to 9.6.9
- Add DBFamilyMap
- Add RDSDBClusterParameterGroup

*Issue #, if available:*
Resolves:  https://github.com/aws-quickstart/quickstart-amazon-aurora/issues/18

*Description of changes:*
Make the Postgres Database Engine Selectable

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: templates/aurora_postgres-master.template.yaml [hunk 1]
```diff
@@ -183,6 +186,14 @@ Parameters:
     Default: "35"
     Description: "The number of days for which automatic database snapshots are retained."
     Type: String
+  DBEngineVersion:
+    Description: Select Database Engine Version
+    Type: String
+    Default: 9.6.9
+    AllowedValues:
```
Comment (bpar476) [inline templates/aurora_postgres-master.template.yaml]: Is there a reason 9.6.11 is not available?

Comment (avattathil) [reply templates/aurora_postgres-master.template.yaml]: Will add it

**Comment by bpar476** [comment]
Do you think we could use the default parameter group for each postgres version? Or make the `DBClusterParameterGroup` a parameter that can be overridden if you want `force_ssl` to be the default?

**Comment by catcharbind** [comment]
@bpar476 PR was updated to create a `RDSDBClusterParameterGroup` and a `DBParamGroup` based on the PostgreSQL engine version selected so that customers can modify the parameters as per their requirement. `force_ssl` is not turned on by default in the quickstart, which is the default behavior.

**Comment by bpar476** [comment]
@catcharbind It was set before but looks like it has been disabled now. I tested with the force_ssl flag off and it all seems to work fine with our app 👍

**Pull request merged**